src/components/home/modals/ColdSignature.vue

```diff
@@ -99,7 +99,8 @@ export default {
                 background: '#ffffff',
                 foreground: '#000000'
             }
-            const imgBase64 = jrQrcode.getQrBase64(JSON.stringify(this.dataObject), options)
+            const text = JSON.stringify(this.dataObject).replace(/"amount":"(\d+)"/g, '"amount":$1')
```

Comment:
Why you need to remove the quotes here?

Reply:
When BigNumber serialize to JSON, it will show as string. To convert the amount from string to long, we need  add `replace` here.

Reply:
Ok, I see